app/code/Magento/ImportExport/Test/Unit/Model/Import/AdapterTest.php

```diff
@@ -5,33 +5,157 @@
  */
 namespace Magento\ImportExport\Test\Unit\Model\Import;
 
-use Magento\ImportExport\Model\Import\Adapter as Adapter;
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Filesystem;
+use Magento\ImportExport\Model\Import\Adapter;
+use Magento\ImportExport\Model\Import\Source\FileFactory;
+use Magento\ImportExport\Model\Import\Source\FileParser\CsvParserFactory;
+use Magento\ImportExport\Model\Import\Source\FileParser\ParserFactoryInterface;
+use Magento\ImportExport\Model\Import\Source\FileParser\ZipParserFactory;
+use Magento\ImportExport\Test\Unit\Model\Import\Source\FileParser\FakeObjectManager;
+use Magento\ImportExport\Test\Unit\Model\Import\Source\FileParser\FakeParser;
+use Magento\ImportExport\Test\Unit\Model\Import\Source\FileParser\FakeParserFactory;
```

Comment:
This is not Unit test in terms of Magento 2,
This is an Integration test. Think we need to keep consistency in our approaches